x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -400,30 +489,61 @@ export const ModelsList: FC<Props> = ({
       'data-test-subj': 'mlModelsTableRowDetailsToggle',
     },
     {
-      field: ModelsTableToConfigMapping.id,
-      name: i18n.translate('xpack.ml.trainedModels.modelsList.modelIdHeader', {
-        defaultMessage: 'ID',
-      }),
-      sortable: true,
+      name: modelIdColumnName,
+      width: '15%',
+      sortable: ({ model_id: modelId }: ModelItem) => modelId,
       truncateText: false,
+      textOnly: false,
       'data-test-subj': 'mlModelsTableColumnId',
+      render: ({ description, model_id: modelId }: ModelItem) => {
+        const isTechPreview = description?.includes('(Tech Preview)');
+
+        return (
+          <EuiFlexGroup gutterSize={'s'} alignItems={'center'} wrap={true}>
+            <EuiFlexItem grow={false}>{modelId}</EuiFlexItem>
+            {isTechPreview ? (
+              <EuiFlexItem grow={false}>
+                <TechnicalPreviewBadge compressed />
+              </EuiFlexItem>
+            ) : null}
+          </EuiFlexGroup>
+        );
+      },
     },
     {
-      field: ModelsTableToConfigMapping.description,
-      width: '350px',
+      width: '35%',
       name: i18n.translate('xpack.ml.trainedModels.modelsList.modelDescriptionHeader', {
         defaultMessage: 'Description',
       }),
-      sortable: false,
       truncateText: false,
       'data-test-subj': 'mlModelsTableColumnDescription',
-      render: (description: string) => {
+      render: ({ description, recommended }: ModelItem) => {
         if (!description) return null;
-        const isTechPreview = description.includes('(Tech Preview)');
         return (
           <>
             {description.replace('(Tech Preview)', '')}
-            {isTechPreview ? <TechnicalPreviewBadge compressed /> : null}
+            {recommended ? (
+              <EuiToolTip
+                content={
+                  <FormattedMessage
+                    id="xpack.ml.trainedModels.modelsList.recommendedDownloadContent"
+                    defaultMessage="Recommended model version for your cluster's hardware configuration"
```

Comment:
I think we should we add `ELSER` into this - `Recommended ELSER model version for your cluster's hardware configuration`

Reply:
And as discussed - if you can retain the `recommended`  label after the model has been downloaded, that would be a nice-to-have.

Reply:
Updated by https://github.com/elastic/kibana/pull/167407/commits/efe9c0d10322c57836ccada752425efa04f2d113 and https://github.com/elastic/kibana/pull/167407/commits/83f4d59f599e6583957de6c262e6ea0f5869fd22